2.4-taptree.ipynb

```diff
@@ -6,14 +6,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import hashlib\n",
-    "from io import BytesIO\n",
-    "\n",
     "import util\n",
     "from test_framework.address import program_to_witness\n",
-    "from test_framework.key import ECKey, ECPubKey, generate_key_pair\n",
-    "from test_framework.messages import CTransaction, COutPoint, CTxIn, CTxOut, CScriptWitness, CTxInWitness, ser_string, sha256\n",
-    "from test_framework.script import TapTree, TapLeaf, Node, CScript, OP_1, TaprootSignatureHash, OP_CHECKSIG, SIGHASH_ALL_TAPROOT"
+    "from test_framework.key import ECPubKey, generate_key_pair\n",
+    "from test_framework.messages import CTxInWitness, ser_string, sha256\n",
```

Comment:
We need CScriptWitness for Programming Exercise 2.4.9

Reply:
fixed